src/components/ObsEdit/CVSuggestions.js

```diff
@@ -127,12 +125,20 @@ const CVSuggestions = ( ): Node => {
     />
   );
 
+  const displayPhotos = ( ) => {
+    return currentObs.observationPhotos.map( p => {
+      return {
+        uri: p.photo?.url || p?.photo?.localFilePath
+      };
+    } );
+  };
+
   return (
     <ViewNoFooter>
       <View>
         {hasPhotos && (
-          <EvidenceList
-            currentObs={currentObs}
+          <PhotoCarousel
+            photos={displayPhotos( )}
             setSelectedPhoto={setSelectedPhoto}
```

Comment:
This needs to become `setSelectedPhotoIndex` now, right? Maybe we should add a flow pre-commit hook.